packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -408,7 +408,8 @@ function TokensPanel({
     isParentChainArbitrumSepolia,
     isOrbitChain,
     nativeCurrency,
-    getBalance
+    getBalance,
+    walletAddress // required for re-rendering the row when wallet address changes / wallet is connected
```

Comment:
We use Virtualization in Token rows, which need to be manually regenerated if data changes. Adding `walletAddress` to the dependency ensures that.